Apsitvarkom.UnitTests/Models/Validation/PollutedLocationValidationTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Apsitvarkom.Models;
+using FluentValidation;
+
+namespace Apsitvarkom.UnitTests.Models.Validation
```

Comment:
The more common practice in this solution has been using namespace with the semicolon instead of having curly brackets. Maybe let's change it to match it and use
`namespace Apsitvarkom.UnitTests.Models.Validation;`? If you decide to do so, please also match the rest of the indentation.